editor/connections_dialog.cpp

```diff
@@ -1543,6 +1557,7 @@ ConnectionsDock::ConnectionsDock() {
 	tree->connect("item_selected", callable_mp(this, &ConnectionsDock::_tree_item_selected));
 	tree->connect("item_activated", callable_mp(this, &ConnectionsDock::_tree_item_activated));
 	tree->connect("gui_input", callable_mp(this, &ConnectionsDock::_rmb_pressed));
+	tree->connect("gui_input", callable_mp(this, &ConnectionsDock::_connections_gui_input));
```

Comment:
Connecting a single signal to multiple functions looks a bit strange to me. Do we use this pattern frequently elsewhere?

Reply:
`_rmb_pressed` code should be moved to gui input method.

Reply:
Fixed